README.md

```diff
@@ -71,7 +71,7 @@ Amendments to this constitution after ratification can be proposed by any member
 # Article XI
 ## Dissolution (Distribution of Assets and Intellectual Property)
 
-Should the organization be dissolved, assets and intellectual property will be donated to Clemson Computing and Information Technology of Clemson University.
+Dissolution will occur when there are not enough members to have a quorum or by a 75% vote of all members, officers, and advisors. Should the organization be dissolved, assets and intellectual property will be donated to Clemson Computing and Information Technology of Clemson University.
```

Comment:
You've solved one undefined term with a replacement of another undefined term.  Now you should define what a quorum is and when it is measured.

Reply:
Is Article VIII insufficient? https://github.com/CUCyber/constitution/blob/master/README.md#article-viii

Reply:
Not quite,  You've answered the question of how many members are required, but you've not answered of how long a lack of a quorum must exist to trigger the provision.  I.E. you wouldn't want to dissolve the club if only 3 members could make a meeting during the summer, during exam week, or if most the team is say competing elsewhere.

Reply:
"when there are not enough members" - I guess I read that as anytime the number of members recorded by the club is below 4 or does not include an officer or advisor. Even if people are at a competition or it is over the summer they are still members as I read it. Do you have a suggestion for either adjusting that idea or aligning the wording to fit that idea?